core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss

```diff
@@ -29,6 +29,15 @@ p-avatar {
     }
 }
 
+::ng-deep {
+    .p-menu {
+        margin-top: $spacing-0;
+        &.p-menu-overlay {
+            width: 17rem;
+        }
+    }
+}
+
```

Comment:
This is global and affects every `p-menu`, wrap this css rule inside the `:host` pseudo-selector